wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/swervebot/SwerveModule.java

```diff
@@ -113,6 +113,10 @@ public void setDesiredState(SwerveModuleState desiredState) {
     SwerveModuleState state =
         SwerveModuleState.optimize(desiredState, new Rotation2d(m_turningEncoder.getDistance()));
```

Comment:
Add `var encoderRotation = new Rotation2d(m_turningEncoder.getDistance());` and use it in both places.